app/src/androidTest/java/ch/epfl/sweng/SDP/game/drawing/DrawingOfflineWithItemsTest.java

```diff
@@ -29,31 +28,32 @@
 import static org.hamcrest.Matchers.greaterThanOrEqualTo;
 import static org.junit.Assert.assertThat;
 
-public class DrawingOfflineItemsTest {
+public class DrawingOfflineWithItemsTest {
```

Comment:
You should probably change the name to DrawingONLINEWithItemsTest.

Reply:
These are for the practice mode (DrawingOffline class) when enabling the items